src/components/StaticFilters.tsx

```diff
@@ -0,0 +1,133 @@
+import { useMemo } from 'react';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import {
+  ApplyFiltersButton,
+  CheckboxOption,
+  CollapsibleLabel,
+  CollapsibleSection,
+  FilterGroup,
+  SearchInput
+} from './Filters';
+import { StaticFilters as StaticFiltersCompoundComponent } from './Filters/StaticFilters';
+
+/**
+ * The CSS class interface for {@link StaticFilters}.
+ *
+ * @public
+ */
+export interface StaticFiltersCssClasses {
+  container?: string,
+  searchInput?: string,
+  optionsContainer?: string,
+  option?: string,
+  optionInput?: string,
+  optionLabel?: string,
+  applyFiltersButton?: string
+}
+
+/**
+ * The configuration data for a filter option.
+ *
+ * @public
+ */
+export interface FilterOptionConfig {
+  /** {@inheritDoc Filters.CheckboxOptionProps.value} */
+  value: string | number | boolean,
+  /** {@inheritDoc Filters.CheckboxOptionProps.label} */
+  label?: string,
+  /** {@inheritDoc Filters.CheckboxOptionProps.selectedByDefault} */
+  selectedByDefault?: boolean
+}
+
+/**
+ * Props for the {@link StaticFilters} component.
+ *
+ * @public
+ */
+export interface StaticFiltersProps {
+  /** The fieldId corresponding to the static filter group. */
+  fieldId: string,
+  /** {@inheritDoc FilterOptionConfig} */
+  filterOptions: FilterOptionConfig[],
+  /** The displayed label for the static filter group. */
+  title?: string,
+  /** Whether or not the filter is collapsible. Defaults to true. */
+  collapsible?: boolean,
+  /**
+   * If the filter group is collapsible, whether or not it should start out
+   * expanded. Defaults to true.
+   */
+  defaultExpanded?: boolean,
+  /** Whether or not to display a text input to search for filter options. */
+  searchable?: boolean,
+  /**
+   * Whether or not a search is automatically run when a filter is selected.
+   * Defaults to true.
+   */
+  searchOnChange?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: StaticFiltersCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays a group of user-configured filters that will be
+ * applied to the current vertical search.
+ *
+ * @param props - {@link StaticFiltersProps}
+ * @returns A React component for static filters
+ *
+ * @public
+ */
+export function StaticFilters({
+  fieldId,
+  filterOptions,
+  title,
+  collapsible = true,
+  defaultExpanded = true,
+  searchable,
+  searchOnChange = true,
+  customCssClasses = {},
+  cssCompositionMethod
+}: StaticFiltersProps): JSX.Element {
+  const cssClasses = useMemo(() => {
+    return {
+      ...customCssClasses,
```

Comment:
Could we remove these additional spreads and have this be:
```ts
return {
  ...customCssClasses,
  optionContainer: customCssClasses.option,
  label: customCssClasses.optionLabel,
  input: customCssClasses.optionInput
}
```
Also, is it intentional that this memoized `cssClasses` has both the original keys from customCssClasses and also the renamed keys of optionContainer, label, and input? That is that it has the shape of
```ts
{
  // custom css classes shape
  container?: string,
  searchInput?: string,
  optionsContainer?: string,
  option?: string,
  optionInput?: string,
  optionLabel?: string,
  applyFiltersButton?: string,
  
  // renamed keys
  optionContainer: string,
  label: string,
  input: string
}
```

Reply:
This results in some extra keys that I think could lead to confusing behavior in the future.

Reply:
I had tried it without the additional spreads initially, but I noticed that it would pass in 
```ts
{
  optionContainer: undefined,
  label: undefined,
  input: undefined
}
```
as the `customCssClasses` to `CheckboxOption`. Since it explicitly sets those classes as undefined, all the styling for those classes was removed, regardless of composition method. So, I added the additional spreads to make sure the classes weren't set (even as undefined) when the classes aren't defined in the props.

I can update to explicitly list the CSS classes so the extra keys aren't included.

Reply:
Ah I see, that's unfortunate but also makes sense